package.json

```diff
@@ -10,6 +10,7 @@
     "react": "^16.6.3",
     "react-dom": "^16.6.3",
     "react-select": "^2.1.1",
+    "react-redux": "^5.0.7",
```

Comment:
does that mean that every project written with RDS will have to use redux?

Reply:
Nope, it's just a peerDependency - yarn/npm will complain about missing dependency (just like now npm complains that design-system:
```
warning " > design-system@14.2.0" has unmet peer dependency "i18next@10.5.0".
warning "design-system > ember-fetch > rollup-plugin-babel@3.0.7" has incorrect peer dependency "babel-core@6".
```

The `react-redux` is listed as a dependency in README for BannerNotification.